src/main/java/uk/gov/hmcts/ccd/domain/service/aggregated/MergeDataToSearchResultOperation.java

```diff
@@ -136,8 +136,10 @@ private SearchResultViewItem buildSearchResultViewItem(final CaseDetails caseDet
         searchResult.getFieldsWithPaths().forEach(searchResultField -> {
             JsonNode jsonNode = caseData.get(searchResultField.getCaseFieldId());
             if (jsonNode != null) {
-                newResults.put(searchResultField.getCaseFieldId() + "." + searchResultField.getCaseFieldPath(),
-                    getObjectByPath(searchResultField, jsonNode));
+                Object objectByPath = getObjectByPath(searchResultField, jsonNode);
+                if (objectByPath != null) {
+                    newResults.put(searchResultField.getCaseFieldId() + "." + searchResultField.getCaseFieldPath(), objectByPath);
```

Comment:
```suggestion
                    newResults.put(searchResultField.buildCaseFieldId(), objectByPath);
```

Reply:
That's and existing logic, we can look into this later